Update Custom Properties Page (#861)
DevCycleHQ/devcycle-docs: docs/platform/feature-flags/targeting/custom-properties.md

Then, finish your Targeting Rule by filling out the comparator and the value fields. 

There are no limits to the number of Properties that can be used to define a single Targeting Rule! Use as many custom properties as necessary to accurately target your users for your Features. 

![property type open](/feb-2022-custom-properties-in-targeting-rules.png)

## SDK Usage

See the [Identify](/sdk/features#identifying-a-user-or-setting-properties) Documentation for the usage of properties within each SDK.

## Managing Custom Properties

To manage the Custom Properties in a project, navigate to the Settings page and click on "Custom Properties". This will display a list of Custom Properties that you've created for the project where you can filter by project or custom property type.

![image of custom properties settings](/custom-properties-settings.png)

> To make edits to a Custom Property, click into the Custom Property or "View Info". You can edit the Custom Properties attributes, setup a schema, gain insight into where it's being used, and delete the Custom Property on this page.

Reviewer: ```suggestion
To make edits to a Custom Property, click into the Custom Property or "View Info". You can edit the Custom Property's attributes, setup a schema, gain insight into where it's being used, and delete the Custom Property on this page.
```

![image of custom properties info page](/custom-properties-manage-page.png)

:::caution

If the **Property Key** is changed, any Identify calls or user objects which are using the previous key will no longer apply to this custom property.

If a Custom Property is **deleted** while in use in a Targeting Rule, the targeting rule will continue to function as normal. 
This custom property will no longer be selectable for new targeting rule definitions.

:::

### Custom Property Schemas

> Custom Property Schemas let you to define allowable values for your Custom Properties when adding them to Targeting Rules. They ensure that only values that are acceptable are added to Targeting Rules, minimizing the risk of setting up a Feature or a Targeting Rule incorrectly.

Reviewer: ```suggestion
Custom Property Schemas let you define allowable values for your Custom Properties when using them in Targeting Rules. Schemas ensure that only values that are acceptable are added to Targeting Rules, minimizing the risk of setting up a Feature or a Targeting Rule incorrectly.
```
